tests/lib/external/snapd-testing-tools/utils/spread-manager

```diff
@@ -135,31 +135,15 @@ main() {
 
     local subcommand="$1"
     local action=
-    local params=
     while [ $# -gt 0 ]; do
```

Comment:
Do we still need this `while`?  We know that `$#` greater than 0 at this point. And we will break or exit in any case.

Reply:
done